### PR TITLE
DECK Plan CR- New SPOT EQUIPMENT  elements for DECK Plan CR

### PR DESCRIPTION
**Pull request by nick-knowles**
Prerequisite components  for Deck Plan chanegs.  Example will be provided with Deck plan

### CODE REVIEW

**Comment by skinkie** [comment]
Good job @nick-knowles this is how we like them.

File: xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element name="SeatBackHeight" type="LengthType" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Height of Seat back from seat bottom.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="SeatDepth" type="LengthType" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Depgth of Seat   from front to back.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd:218]:
```suggestion
					<xsd:documentation>Depth of Seat from front to back.</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_support.xsd [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+		<xsd:restriction base="SpotEquipmentIdType"/>
+	</xsd:simpleType>
+	<xsd:element name="SeatEquipmentRef" type="SeatEquipmentRefStructure" substitutionGroup="SpotEquipmentRef">
+		<xsd:annotation>
+			<xsd:documentation>Identifier of a SEAT EQUIPMENT. +v2.0</xsd:documentation>
+		</xsd:annotation>
+	</xsd:element>
+	<xsd:complexType name="SeatEquipmentRefStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for a reference to a  SEAT EQUIPMENT.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_support.xsd:99]:
```suggestion
			<xsd:documentation>Type for a reference to a SEAT EQUIPMENT.</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="IsFoldup" type="xsd:boolean" default="false" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether seat folds up. default is false</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="IsReclining" type="xsd:boolean" default="false" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether seat reclines. Deafult sis false.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd:228]:
```suggestion
					<xsd:documentation>Whether seat reclines. Default is false.</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:group name="BedEquipmentGroup">
+		<xsd:annotation>
+			<xsd:documentation>Elements for BED EQUIPMENT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element name="Length" type="LengthType" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Length of bedt back from seat bottom.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd:286]:
```suggestion
					<xsd:documentation>Length of bed back from seat bottom.</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd [hunk 5]
```diff
@@ -0,0 +1,359 @@
...
+				<Type>Standard</Type>
+			</Metadata>
+		</xsd:appinfo>
+		<xsd:documentation>SPOT EQUIPMENT types for NeTEx.</xsd:documentation>
+	</xsd:annotation>
+	<!-- === SPOT EQUIPMENT ============================================ -->
+	<xsd:element name="SpotEquipment_" substitutionGroup="InstalledEquipment">
+		<xsd:annotation>
+			<xsd:documentation>An abstract EQUIPMENT in a LOCATABLE SPOT providing a onboard seat, bed or other amenity. +V2.0
+</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd:65]:
```suggestion
			<xsd:documentation>An abstract EQUIPMENT in a LOCATABLE SPOT providing a onboard seat, bed or other amenity. +V2.0</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd [hunk 6]
```diff
@@ -0,0 +1,359 @@
...
+						</xsd:annotation>
+					</xsd:attribute>
+				</xsd:restriction>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	<xsd:element name="SpotEquipment" substitutionGroup="InstalledEquipment">
+		<xsd:annotation>
+			<xsd:documentation>An abstract EQUIPMENT in a LOCATABLE SPOT providing a onboard seat, bed or other amenity. +V2.0
+</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd:93]:
```suggestion
			<xsd:documentation>An abstract EQUIPMENT in a LOCATABLE SPOT providing a onboard seat, bed or other amenity. +V2.0</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd [hunk 7]
```diff
@@ -0,0 +1,357 @@
...
+			</xsd:element>
+			<xsd:element name="UsbPowerSocket" type="xsd:boolean" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether Spot has UsbPowerSockets.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+		</xsd:sequence>
+	</xsd:group>
+	<!-- === SEAT EQUIPMENT ============================================ -->
+	<xsd:element name="SeatEquipment" substitutionGroup="InstalledEquipment">
```
Comment (Aurige) [inline xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd:159]: As we already have a "SeatingEquipment" (with a different purpose) with attributes like _Armrest_ and _SeatHeight_ , shouldn't we harmonise and also clarify when to use each other in the annotation ?

File: xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd [hunk 8]
```diff
@@ -0,0 +1,357 @@
...
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="SpotEquipment_VersionStructure">
+				<xsd:sequence>
+					<xsd:group ref="BedEquipmentGroup"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:group name="BedEquipmentGroup">
```
Comment (Aurige) [inline xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd:277]: Shouldn't we have a "Level" (ground, one up, and maybe two up) and maybe some info on the climbing device (but that's probably always a ladder) and space above ?

**Comment by ue71603** [comment]
@Aurige  Who do you address the question to? If this should be done, e.g. @trurlurl can do an update (well, as also this one is from a private repository, we would have to create a new one)

File: xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd (hunk 8 above)
Comment (nick-knowles) [reply xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd:277]: YEs we could add BunkLevel   bottom/first/top l and shoudl also add headroom

**Comment by ue71603** [comment]
done in #615

**Pull request closed without merging**